src/main/resources/templates/apidocumentation/apiDocumentationForTag.ftl

```diff
@@ -0,0 +1,9 @@
+<#include '../layout.ftl'>
+
+<@defaultPage pageHeading="${tagName}">
+  <#list operationList as operation>
+    <@govukDetails.details summaryTitle="${operation.getSummary()}">
```

Comment:
You have to use the java method signatures as there are `summary()` and 'description()' methods in the Operation class